fastapi_security/api.py

```diff
@@ -26,6 +31,7 @@ class FastAPISecurity:
 
     def __init__(self, *, user_permission_class: Type[UserPermission] = UserPermission):
         self.basic_auth = BasicAuthValidator()
+        self.basic_auth_with_digest = BasicAuthWithDigestValidator()
```

Comment:
I think it gets kind of messy when we add another validator to the `FastAPISecurity` class. I would suggest that you add `basic_auth` as an optional keyword argument to the class instead. If it's not passed in you default to initializing `BasicAuthValidator` in the constructor.
```suggestion

```

Reply:
:+1: